company_website/static/how_we_work/partials/time_materials_fixed_price_graph.sass

```diff
@@ -306,6 +306,9 @@
 							color: $graph-red-color
 							border-color: $graph-red-color
 
+							@media only screen and (max-width: 430px)
+								padding: 5px 0
+
```

Comment:
@pawelkisielewicz  this is not a right screen size and it is not fix bug, this solution gave another one bug under 430px, the problem I described was beetwen 928px and 850px.